_gsocblogs/2025/blog_ATLAS_YolanneLee.md

```diff
@@ -0,0 +1,103 @@
+---
+project: ATLAS
+title: Neural (De)
+author: Jane Doe
+photo: blog_authors/JaneDoe.jpg   # Upload your square photo here
+avatar: https://avatars.githubusercontent.com/u/12345678?s=400&v=4   # Replace with your GitHub avatar URL
+date: 31.08.2025   # Use the date you wrote the article
+year: 2025
+layout: blog_post
+logo: hsf_logo_angled.png   # Match the logo file listed in your project’s metadata
+intro: |
+  In high-energy physics experiments such as those at CERN’s ATLAS project, immense volumes of data are generated. This project explores the feasability for “precision upsampling” using deep generative models to be used to reconstruct high-precision floating-point data from aggressively compressed representations.
+---
+
+|  |  |
+| --- | --- |
+| Name | [Yolanne Lee](https://maths4dl.ac.uk/team-member/yolanne-lee) |
+| Organisation | [CERN ATLAS Project](https://atlas.cern/) |
+| Mentor | [Dr. Maciej Szymański](https://www.anl.gov/profile/maciej-pawel-szymanski), [Dr. Peter Van Gemmeren](https://www.anl.gov/profile/maciej-pawel-szymanski)|
+| Project | [Neural (De)compression for High Energy Physics](https://summerofcode.withgoogle.com/programs/2025/projects/W5UjfwMc) |
+
+## Introduction
+
+In high-energy physics experiments such as those at CERN’s ATLAS project, immense volumes of data are generated. This project explores the feasability for “precision upsampling” using deep generative models to be used to reconstruct high-precision floating-point data from aggressively compressed representations. I had the opportunity to work on this topic with the support and supervision of Maciej Szymański and Peter Van Gemmeren with the ATLAS Software & Computing group.
```

Comment:
Please consider adding the references to e.g. CERN, ATLAS Open Data, IEEE 754, ML models, software you used, etc. Either inline links or collectively at the end, as you prefer.